CMakeLists.txt

```diff
@@ -2626,9 +2622,9 @@ endif()
 
 # WavPack audio file support
 find_package(wavpack)
-option(WAVPACK "WavPack audio file support" OFF)
+cmake_dependent_option(BULK "WavPack audio file support" ON "wavpack_FOUND" OFF)
```

Comment:
I still think this is a bad idea for the reasons I outlined here: https://github.com/mixxxdj/mixxx/pull/3707#discussion_r595508414

I think we should stop using `cmake_dependent_option` and use a regular option instead.

Reply:
This only establishes consistency. If we want to change it we should change the behavior of **all** feature options **at once**.

Reply:
Ok

Reply:
Inconsistent behavior is the worst of all ;)